inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -237,3 +276,18 @@ def get_circular_curve_coefficients(toroidal_angle, major_radius, minor_radius):
     curve_coefficients[4, 1] = -minor_radius
 
     return curve_coefficients
+
+
+@StellaratorCoils.create_input_files.register
+def _(self, simulator: Simsopt, input_dir):  # pylint: disable=unused-argument
+    """Creates Simsopt simulation input files."""
+
+    coil_coefficients = [coil.curve_coefficients for coil in self.coils]
+    coil_currents = np.array([coil.current for coil in self.coils])
```

Comment:
Why are you converting one to `np.array` and not the other?

Reply:
Because coil_coefficients is a list of numpy arrays which I can immediately write to a file with numpy.savez whereas coil_currents is a list with the currents (list of floats), because each object of the `Coil` class has curve_coefficients, which is a numpy array, and the current which is merely a float.

Reply:
Got it thanks.